apps/mobile/src/components/browser/browser-in-use.tsx

```diff
@@ -1,31 +1,39 @@
 import { useRef, useState } from 'react';
 import { View } from 'react-native';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
-import { WebView, WebViewNavigation } from 'react-native-webview';
+import { WebView, WebViewMessageEvent, WebViewNavigation } from 'react-native-webview';
 
 import ArrowRotateClockwise from '@/assets/arrow-rotate-clockwise.svg';
 import ChevronLeft from '@/assets/chevron-left.svg';
 import ChevronRight from '@/assets/chevron-right.svg';
 import Cross from '@/assets/cross-large.svg';
 import DotGridHorizontal from '@/assets/dot-grid-horizontal.svg';
 import { Modal } from '@/components/bottom-sheet-modal';
+import injectedProvider from '@/scripts/injected-provider';
 import { BottomSheetModal } from '@gorhom/bottom-sheet';
 import { useTheme } from '@shopify/restyle';
 
 import { Box, Text, Theme, TouchableOpacity } from '@leather.io/ui/native';
 
+import { ApprovalUXModal } from './approval-ux-modal';
+
 interface BrowserInUseProp {
   textURL: string;
   goToInactiveBrowser: () => void;
 }
 
+export type BrowserMessage = {
+  jsonrpc: string;
+  id: string;
+  method: 'getAddresses';
+} | null;
```

Comment:
```suggestion
interface BrowserMessageDetails {
  jsonrpc: string;
  id: string;
  method: 'getAddresses';
}
export type BrowserMessage = BrowserMessageDetails | null
```

may be more friendly to define this way in case we need to refer to the non-nullable type?